go.mod

```diff
@@ -192,3 +192,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/compose-spec/compose-go/v2 => github.com/glours/compose-go/v2 v2.0.0-20241023174721-0f71bc7922c4
```

Comment:
you can point to compose-go HEAD as your PR was merged

Reply:
Done!